src/main/java/reposense/util/FileUtil.java

```diff
@@ -104,9 +104,18 @@ public static void zip(Path sourcePath, Path outputPath, String... fileTypes) {
      * Unzips the contents of the {@code zipSourcePath} into {@code outputPath}.
      */
     public static void unzip(Path zipSourcePath, Path outputPath) {
-        ZipEntry entry;
         try (
```

Comment:
no need for line break when only 1 statement is within the try-with
Make it something like:
```
try(InputStream is = Files.newInputStream(zipSourcePath)) {
} 
```